helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -1,12 +1,31 @@
 ## nginx configuration
-## Ref: https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md
+## Ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/index.md
 ##
+
 controller:
+  name: controller
```

Comment:
All the values that are the same as upstream doesn't need to be here. We rather have this file as minimal as possible and rely on upstream values to be "good" values
So remove all the values that we don't need to have here.

Reply:
should I also remove the comments?

Reply:
I'd say yes, but leave the comments that are referencing a variable that we need to have here.
But all the comments that are for a variable that aren't in this file should be removed.

Reply:
done